catalogue/webapp/test/e2e/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { baseUrl } from './urls';
+
+export type toggleFeatureProps = {
```

Comment:
Is this a cookie type?

Reply:
yeah, do we have something available like this ? or Should i ideally get it from DefinitelyTyped ?
https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/cookies